skipper.yaml

```diff
@@ -11,7 +11,6 @@ volumes:
   - $MINIKUBE_HOME:$MINIKUBE_HOME
   - $HOME/.minikube:$HOME/.minikube
   - $HOME/.kube/:$HOME/.kube
-  - $HOME/.docker/:$HOME/.docker
```

Comment:
This would've been replaced by skipper new behaviour that it's mounting by default the docker folder (the same mount as we are removing in this PR, that would've been double mounted and broken CI and local env) However we decided to pin the version instead, so this PR is deprecated - closing